.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       matrix:
         target: [macos-universal-10.15]
-    runs-on: macos-11
+    runs-on: macos-12
```

Comment:
this needs to be 13, github and apple and deprecating old versions quite fast

Reply:
13 it is.